pippo-demo/src/main/java/ro/fortsoft/pippo/demo/SimpleApplication.java

```diff
@@ -122,7 +138,7 @@ public void handle(Request request, Response response, RouteHandlerChain chain)
 
                 Map<String, Object> model = new HashMap<>();
                 model.put("greeting", message);
-                response.render("hello", model);
+                response.html().render("hello", model);
```

Comment:
The same comment. It's more direct `response.render()` but as long as both are OK who cares :)